### ISSUE TITLE
finetune results on UCF101

### ISSUE THREAD

**Issue by thematrixduo** [antoine77340/S3D_HowTo100M#7]
Hi, I am having some trouble reproducing the finetune resuts on UCF101. I can only get 88.6 while the reported result is 91.3. Could you share your finetune script or procedure/hyper-params? Thanks!

**Comment by antoine77340** [comment]
Hi,

Yes the fine-tuning is a bit tricky and involves finding the right set of hyper-parameters as well as right data augmentations. 
However, this codes relies on many internal tools from DeepMind so I won't be able to share them with you unfortunately :(.

I would be however happy to clarify any aspect of the fine-tuning if you have any questions.

**Comment by thematrixduo** [comment]
Thanks for the reply. I would appreciate if you can let me know your hyper-params (batch size, learning rate, dropout rate...) and how do you process the data when testing (center-crop? 5-crop? etc).

**Comment by jalayrac** [comment]
Hi,

We follow the same procedure as in this paper: https://arxiv.org/pdf/2006.16228.pdf. The finetuning procedure is detailed in the appendix and I will copy it here for reference.

> FT on UCF101/HMDB51. For fine-tuning, we use the SGD optimizer with momentum = 0.9. A
learning rate schedule is used where the learning rate gets multiplied by γ at the given steps (values
for each dataset are provided in Table 5). We also apply weight decay to the variables of the linear
classifier. Because in FT, the network can readapt to slight changes in the input, we resize the
minimum side to 256 and take random crops of size 256 × 256. At test time, we take 10 linearly
spaced clips and average their predictions to get the final score. We take the central crops of the
frames of size 256 × 256. We do not apply color augmentation, scale jittering or horizontal flipping
during test time.

The Table 5 gives the hyperparameters for HMDB51/UCF101.

Parameter HMDB51 UCF101
LR base 1.0 1.0
LR decay γ 0.1 0.1
LR schedule 2.5K/5K/7.5K 6.25K/12.5K/18.75K
Weight decay 5 ∗ 10−3 10−7
Batch size 256 256
Training steps 10K 25K

You might want to sweep a bit the initial learning rate (e.g. between [0.01, 0.1, 1]) as well as the weight decay. However I would expect that the above value should give you something reasonable.

**Comment by jalayrac** [comment]
No dropout is employed.

**Comment by thematrixduo** [comment]
That is pretty clear. Thanks a lot!
